CMakeLists.txt

```diff
@@ -197,7 +202,14 @@ elseif(PLATFORM_LINUX)
     set(GL_SUPPORTED       TRUE CACHE INTERNAL "OpenGL is supported on Linux platform")
     set(VULKAN_SUPPORTED   TRUE CACHE INTERNAL "Vulkan is supported on Linux platform")
     set(ARCHIVER_SUPPORTED TRUE CACHE INTERNAL "Archiver is supported on Linux platform")
-    target_compile_definitions(Diligent-PublicBuildSettings INTERFACE PLATFORM_LINUX=1)
+    if(PLATFORM_FREEBSD)
+        target_compile_definitions(Diligent-PublicBuildSettings INTERFACE PLATFORM_LINUX=1 PLATFORM_FREEBSD=1)
+        if(EXISTS /usr/local/include)
+            include_directories(/usr/local/include)
```

Comment:
Diligent's build system does not use global CMake include directories.
All directories are set for targets.

Which include files from `/usr/local/include` are need to be used?

Reply:
All the X, Wayland, GL, and Vulcan stuffs, since our base is pretty small.  The vast majority of everything is installed manually.

Reply:
Diligent uses its own headers for Vulkan. For GL, does `find_package` not provide all the required libraries?
We need to find another solution. Heaving global `include_directories(/usr/local/include)` here and in multiple other places is not a good option.

Reply:
CMake's `find_package` finds the libraries just fine (needed for linker), but it won't provide headers.  If CMake provides a way to specify a search location to the compiler for specific headers, I don't know about it, and I would use that instead.